user-interface/src/main/java/life/qbic/datamanager/files/parsing/converters/MetadataConverterV2.java

```diff
@@ -0,0 +1,32 @@
+package life.qbic.datamanager.files.parsing.converters;
+
+import java.util.List;
+import life.qbic.datamanager.files.parsing.MetadataConverter;
+import life.qbic.datamanager.files.parsing.ParsingResult;
+
+/**
+ * Metadata Converter V2
+ * <p>
+ * Successor of the {@link MetadataConverter} interface.
+ * <p>
+ * The interface is used to provide a generic way of converting a {@link ParsingResult} into a list
+ * of different metadata types.
+ *
+ * @since 1.10.0
+ */
+public interface MetadataConverterV2<T> {
```

Comment:
```suggestion
@FunctionalInterface
public interface MetadataConverterV2<T> {
```
we only have one abstract method

Reply:
If you dont mind, i would like to defer this request until we actually need this annotation. Once we have the annotation and make use of it, we cant extend this interface anymore. 
Since I don't see the advantage in the current design and usage.